modules/core/src/lib/deck.js

```diff
@@ -525,6 +525,10 @@ export default class Deck {
       _pickRequest.radius = this.props.pickingRadius;
     }
 
+    if (this.layerManager) {
```

Comment:
Since `_requestPick` is no longer just a picking method, maybe it should be renamed.